engine/src/main/scala/cromwell/engine/backend/CromwellBackend.scala

```diff
@@ -16,13 +16,19 @@ import scala.util.Try
   * for now
   */
 object CromwellBackend {
-  private val log = LoggerFactory.getLogger(getClass.getName)
   private var _backends: Option[Map[String, Backend]] = None
   private var _defaultBackend: Option[Backend] = None
 
-  def initBackends(backendTypes: List[String], defaultBackend: String, actorSystem: ActorSystem) = {
-    _backends = Option((backendTypes map { backendType => (backendType, Backend.from(backendType, actorSystem)) }).toMap)
-    _defaultBackend = Option(Backend.from(defaultBackend, actorSystem))
+  def backendFromConfig(entry: BackendConfigurationEntry, actorSystem: ActorSystem): Backend = {
```

Comment:
I wouldn't dispute that, and here it looks like there aren't engine dependencies.